### ISSUE TITLE
Make the link with mySociety more prominent on the mailouts

### ISSUE THREAD

**Issue by MyfanwyNixon** [mysociety/whatdotheyknow-theme#1840]
This is part of a wider push to ensure that users of each of our services know that it is one of many provided by mySociety.

Is it possible to add the 'a service from mySociety' pennant at the top of these mailings? I think they're plain text so might have to think of something else?

- Email alerts
- 'You have a response' emails
- Any other emails that we send in the course of a user transaction (sorry, did try looking at the code but I can't find them here or in Alaveteli!)

If we can't, perhaps we could look at how to reposition
>WhatDoTheyKnow is a project of mySociety run by a small team of staff and dedicated volunteers.

Which at the moment is just right at the bottom, in small text and easy to miss.

**Comment by HelenWDTK** [comment]
Site emails are plain text at the moment, so likely not. 

I think the base content lives in the various mailer bits of the views e.g.

app/views/notification_mailer
app/views/survey_mailer
app/views/track_mailer
app/views/user_mailer

**Comment by garethrees** [comment]
> Site emails are plain text at the moment, so likely not.

Yeah, this. Would need https://github.com/mysociety/alaveteli/issues/3420 (not currently planned through any of the funded work programmes).

> If we can't, perhaps we could look at how to reposition
>> WhatDoTheyKnow is a project of mySociety run by a small team of staff and dedicated volunteers.
> Which at the moment is just right at the bottom, in small text and easy to miss.

https://github.com/mysociety/whatdotheyknow-theme/blob/master/lib/views/general/_default_footer.text.erb is the current footer. I think we could make that a lot terser, which while still in the footer (which I think is appropriate in the context) makes it a bit less noisy that gets glossed over, along the lines of:

```
---

WhatDoTheyKnow is a project of mySociety. Support us at https://www.mysociety.org/donate

How we handle your data: <%= help_privacy_url %>

mySociety is a registered charity in England and Wales (1076346) and a limited
company (03277032)
```

Do we even need the final line?

**Comment by MyfanwyNixon** [comment]
Thanks both! Shame, but understood, about the plain text.

I agree we probably don't need the final line. 

Actually we might be able to wrap it, and the donation request, into one, as @alicewilliams36 has been doing elsewhere:

--
WhatDoTheyKnow is a project of mySociety run by a small team of staff and dedicated volunteers.

mySociety is a charity that believes that information should be accessible to everyone. If this is important to you, please consider donating to help us do more: https://www.mysociety.org/donate

Find out more about how we handle your data at:
<%= help_privacy_url %>

--

I do still wonder how many people get to the end of the message and bother to read this part! Maybe we could consider certain weeks of the year as donation drives and move the donation request temporarily to the top during those times?

**Comment by HelenWDTK** [comment]
We need to include the final line on all "business communications" to comply with the law, so I'd leave it on.

**Comment by HelenWDTK** [comment]
@MyfanwyNixon Is this still something that we want to do?
